tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/TychoFeatureMapping.java

```diff
@@ -33,6 +34,16 @@ public class TychoFeatureMapping extends AbstractXMLTychoMapping {
     private static final String FEATURE_XML = "feature.xml";
     public static final String PACKAGING = "eclipse-feature";
 
+    @Override
+    protected String getPackaging() {
```

Comment:
It seems there is some reordering of code now, can you revert to the original location sow we get (hopefully) a smaller Diff?